src/test/components/__snapshots__/ThreadActivityGraph.test.js.snap

```diff
@@ -1150,10 +1150,6 @@ Array [
     "set fillStyle",
```

Comment:
This snapshot change is expected, because now we don't show samples as "unselected" when there's no selected call node. We show them as if everything that's not filtered out is selected.